draft-ietf-quic-multipath.md

```diff
@@ -124,24 +124,26 @@ path management in {{Section 9 of QUIC-TRANSPORT}} and therefore
 requires negotiation between the two endpoints using a new transport
 parameter, as specified in {{nego}}.
 
-This proposal supports the negotiation of either the use of one packet
-number space for all paths or the use of separate packet number spaces
-per path. While both approaches are supported by the specification in
-this version of the document, the intention for the final publication
-of a multipath extension for QUIC is to choose one option in order to avoid
-incompatibility. More evaluation and implementation experience is needed
-to select one approach before final publication. Some discussion about
-pros and cons can be found here:
-https://github.com/mirjak/draft-lmbdhk-quic-multipath/blob/master/presentations/PacketNumberSpace_s.pdf
-
-As currently defined in this version of the draft the use of multiple
-packet number spaces requires the use of connection IDs is both
-directions. Today's deployments often only use destination connection ID
-when sending packets from the client to the server as this addresses the
-most important use cases for migration, like NAT rebinding or mobility
-events. Further discussion and work is required to evaluate if the use
-of multiple packet number spaces could be supported as well when
-the connection ID is only present in one direction.
+The proposal introduces support for multiple packet number spaces.
+When multipath is negotiated,
+each destination connection ID is linked to a separate packet number space.
+Using multiple packet number spaces enables direct use of the
+loss recovery and congestion control mechanisms defined in
+{{QUIC-RECOVERY}}.
+
+Some deployments of QUIC use zero-length connection ID.
```

Comment:
RFC9000 says the following:

```
  An endpoint SHOULD NOT initiate migration with a peer that has
   requested a zero-length connection ID, because traffic over the new
   path might be trivially linkable to traffic over the old one.
```

I wonder if the above behavior has implications on maintaining multiple paths when  zero-length connection IDs are in used.

Reply:
The RFC 9000 text suggests that only the node that is choosing to use zero length CID will initiate a new path -- typically a client. That seems fine. Third parties will be able to observe packets flowing between client and server as follow:

* path 1: between address 1 and port 1 on client, and server address and server port on server.
* path 2: between address 2 and port 2 on client, and server address and server port on server.

The addresses and ports will be visible whether the client uses null-length CID or not. We may want to say to point out that using different port numbers for each path may remove 16 bits of possible correlation data, but that applies to RFC 9000 just as well.

Reply:
I agree this is OK in typical scenarios. 

I leave it up to you to decide if it is worth to include a mention about this in the draft. Thanks, @huitema.